setup.py

```diff
@@ -20,14 +20,30 @@ def run(self):
 with open(join(os.path.dirname(__file__), 'README.md')) as readme:
     README = readme.read()
 
+
 def load_requirements():
-    return open(join(dirname(__file__), 'requirements/pkg.txt')).readlines()
+    lines = open(join(dirname(__file__), 'requirements/pkg.txt')).readlines()
+    requirements = []
+    for line in lines:
+        if 'https' not in line:
+            requirements.append(line)
+    return requirements
+
+
+def load_dependency_links():
```

Comment:
Why two similar functions with different names?

Reply:
Yes, not big deal. I can make it one and put one boolean argument and check it inside an `if`